tools/models/model_factory.py

```diff
@@ -0,0 +1,168 @@
+import os, sys
+import torch
+import torch.optim as optim
+from inSituML.utilities import MMD_multiscale, fit, load_checkpoint
+from inSituML.ks_models import INNModel
+
+from inSituML.args_transform import MAPPING_TO_LOSS
+from inSituML.encoder_decoder import Encoder
+from inSituML.encoder_decoder import Conv3DDecoder
```

Comment:
Encoder and Decoder class should be declared in model_config (not wholly defined, just imported from model library in general)